common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -756,12 +756,12 @@ private void writeDeferredBufs() throws IOException {
     }
 
     /**
-     * This throws RuntimeException if the number of IOExceptions have exceeded threshold.
+     * This throws IllegalStateException if the number of IOExceptions have exceeded threshold.
```

Comment:
No big deal but this can be ` * @throws IllegalStateException if ...`

Reply:
[4583b22](https://github.com/apache/spark/pull/35923/commits/4583b229c02c39e4ee7a515e68fd2b3cf08b3b64) fix this